packages/qwik/src/core/v2/client/vnode-diff.ts

```diff
@@ -603,8 +600,8 @@ export const vnode_diff = (container: ClientContainer, jsxNode: JSXOutput, vStar
     const srcLength = srcAttrs.length;
     let dstIdx = ElementVNodeProps.PROPS_OFFSET;
     let dstLength = dstAttrs.length;
-    let srcKey: string | null = srcIdx < srcLength ? srcAttrs[srcIdx++] : null;
-    let dstKey: string | null = dstIdx < dstLength ? dstAttrs[dstIdx++] : null;
+    let srcKey = srcIdx < srcLength ? (srcAttrs[srcIdx++] as SsrAttrKey) : null;
```

Comment:
I don't think this change is correct. 

`SsrAttrKey` is SSR only type. `vnode_diff` is Client only. So I think it is wrong to have SSR type here. Maybe create a new type?

Reply:
right, this is client only